client/components/domains/use-my-domain/utilities/get-availability-error-message.js

```diff
@@ -8,7 +8,7 @@ export function getAvailabilityErrorMessage( { availabilityData, domainName, sel
 	const { status, mappable, maintenance_end_time, other_site_domain, other_site_domain_only } =
 		availabilityData;
 
-	if ( domainAvailability.AVAILABLE === status ) {
+	if ( [ status, mappable ].includes( domainAvailability.AVAILABLE ) ) {
```

Comment:
This change means that a domain that is unregistered will always lead the user to search for a domain to register:

![image](https://github.com/user-attachments/assets/8adbb5a8-942f-4a47-9db3-8eefd0e3a556)

Reply:
I was worried that adding the `mappable` value here might have unintended side-effects, but I think the only place where `mappable` can be set to "available" is in the new check added in 170089-ghe-Automattic/wpcom, so it should be fine 🤔